src/model/error.rs

```diff
@@ -111,20 +111,22 @@ impl fmt::Display for Minimum {
 /// use serenity::prelude::*;
 /// use serenity::Error;
 ///
+/// const BAN_DURATION: u32 = 8 * 24 * 60 * 60;
```

Comment:
Can we name it what it is? BAN_DURATION is so generic and forces you to use math to figure out how long this actually is.

Reply:
Renamed to EIGHT_DAYS_IN_SECONDS